isort/identify.py

```diff
@@ -5,7 +5,7 @@
 from pathlib import Path
 from typing import Iterator, NamedTuple, Optional, TextIO, Tuple
 
-from isort.parse import _normalize_line, _strip_syntax, skip_line
+from isort.parse import _strip_syntax, normalize_line, skip_line
```

Comment:
Following the same logic, I would rename `_strip_syntax` to `strip_syntax`. It will be better if you could also do it ;)